co-med-backend-services/app/scripts/app.js

```diff
@@ -0,0 +1,14 @@
+process.env.SUPPRESS_NO_CONFIG_WARNING = 'y';
+const config = require('config');
```

Comment:
The config object is not used. Please remove this.

Reply:
removed config object from app.js